### ISSUE TITLE
`isisdl --compress` doesn't find non-existent video

### ISSUE THREAD

**Issue by MatteoGaetzner** [Emily3403/isisdl#10]
As the title suggests, launching `isisdl --compress` ends up crashing, because a non-existent video doesn't get found.
I've checked isis and found that the video is available for manual downloading there, but isisdl didn't download it or removed it somehow.

**Isisdl output:**
```

Processing ...

Total time: 00:35:06                                  
Total videos: 37 / 312                                
Total time / GB: 950.63s                              
                                                      
Total size before:     73.27 GiB                      
Total size now:        73.27 GiB                      
Total size remaining:  71.31 GiB                      
Total size skipped:     2.18 GiB                      
                                                      
Global efficiency: -89.29%                            
                                                      
Currently processing:                                 
Lecture3Video.mp4                                     
                                                      
Percent done: 99.88%                                  
Finished in:  00:00:00                                
Time elapsed: 00:03:58                                
                                                      
Original  file size:  64.49 MiB                       
Current   file size:  52.90 MiB                       
Estimated file size:  48.92 MiB                       
                                                      
Estimated efficiency: -24.13%                         
Compression score:      1.74                          

I have encountered the following Exception. I'm sorry this happened 😔

Traceback (most recent call last):
  File "/home/matteo/.local/lib/python3.10/site-packages/isisdl/compress.py", line 484, in compress
    compress_status.done_thing(ret_code == 0)
  File "/home/matteo/.local/lib/python3.10/site-packages/isisdl/compress.py", line 223, in done_thing
    new_file_size = os.stat(make_temp_filename(self.cur_file) if was_successful else self.cur_file.path).st_size
FileNotFoundError: [Errno 2] No such file or directory: '/home/matteo/Isis/SoSe2021ML2/Videos/Lecture3Video.mp4'

I have logged this error to the file
/home/matteo/Isis/.errors/1653577444.txt
```

**Comment by Emily3403** [comment]
This is temporarely fixed in version 1.3.9. I'll take a closer look at this problem soon ^^

**Comment by Emily3403** [comment]
Closing due to inactivity
